src/js/components/helpers.js

```diff
@@ -62,10 +62,11 @@ const newText = (type, content, classes = []) => {
  *
  * retorna un nodo de multimedia con su imagen o video y clases asignadas
  */
-const newMultimedia = (type, src, classes = []) => {
+const newMultimedia = (type, src, alt, classes = []) => {
```

Comment:
I don't recommend modify this file in this PR, you could break the components of your classmates.